src/plots/cartesian/axes.js

```diff
@@ -2559,10 +2608,29 @@ axes.drawOne = function(gd, ax, opts) {
             }
         }
 
+        var axDepth = 0;
+        var titleDepth = 0;
+        var multAxisPad = 10; // TODO: Expose as a param to allow user to specify padding between axes
```

Comment:
Wondering could/should we possibly reduce this constant to 3 pixels? 
3 pixel padding used in few other places in plotly.js code so I am just curious it may be a better default here too?

Reply:
3 sounds fine to me! 10 was pretty arbitrary.